.github/workflows/android_release.yml

```diff
@@ -53,17 +53,6 @@ jobs:
         run: |
           git fetch --all --tags -f --depth 1
 
-      - name: Install Qt6 for Linux (required for Android)
```

Comment:
Are you sure removing this works? I couldn't get things to run without it since there were pieces missing. I think the APK file somewhere.

Reply:
The autodesktop option was added after the additional linux qt install, supposedly according to `https://github.com/jurplel/install-qt-action/blob/master/README_upgrade_guide.md` you only need the autodesktop option. And actually I did get ccache working on another branch that I can make a PR for. Instead of 0% hits it is like 90%